### DISCUSSION TITLE
【MockingBird 何去何从】之路线图规划

### DISCUSSION THREAD

**Discussion by babysor** [babysor/MockingBird#622] [General]
This article illustrates the basic roadmap of MockingBird  🐦. If you want to contribute to MockingBird 🐦and you can pick one here to start. Please don't hesitate to ask question here and join our group of enthusiastic developers.
本文主要阐述Mocking Bird🐦的产品路线规划，如果您有想要参与项目贡献，可以在这里找到一个未完成的方向进行。欢迎提出任何问题，并期待一起快乐开发。

除了开发贡献，也欢迎提出产品建议，但请同时体谅，由于这是开源中立项目，有部分建议可能不会被最终采纳或放在高优先级。

### v0.1.0 里程碑 Milestones
---
- [x]  新版GUI Web服务框架选型与实现 https://vaj2fgg8yn.feishu.cn/docs/doccnUWwgIFM6DOi1Y6CuU4QdBc
- [x]  解决GUI Web服务 稳定性与兼容问题 ETA: 2022-06-27
- [x] 解决数字无法被正确发音的bug ETA: 2022-06-27
- [x] 重构代码，为集成CoquiTTS作为后端训练框架准备  ETA: 2022-07-05

### v0.2.0 里程碑 Milestones
---
- [ ] 新模型支持Fast Speech.  ETA: 2022-07-30
- [ ] 新模型支持Fast Pitch.
- [ ] 傻瓜化下载模型，彻底解决模型兼容问题
- [ ] ONNX支持
- [ ] 文档补充
   - [ ] 开发入门指引
   - [ ] 如何使用接口或命令行处理

### 长期规划
---
- [ ] 新合成模型支持VITS.
- [ ] 新Vocoder模型支持.
- [ ] 多语言模型支持.

此外，以下是不同主题的长期交流频道, 同时欢迎使用Github论坛交流：
如何改参数，搞出更逼真的克隆效果 https://github.com/babysor/MockingBird/issues/435
如何改模型，搞出更好效果 https://github.com/babysor/MockingBird/issues/436
训练克隆特定人声音&finetune https://github.com/babysor/MockingBird/issues/437
学术/论文讨论/训练分析 https://github.com/babysor/MockingBird/issues/438
跨语言支持  https://github.com/babysor/MockingBird/issues/440
工程化/新场景讨论（绝不做恶 & 合法合规）https://github.com/babysor/MockingBird/issues/439

**Comment by leo852** [comment]
請問遲下有機會加入廣東話（粵語）嘛？

**Reply by babysor** [reply]
请问有比较明确的需求吗？可以建个discord社区一起交流下，我可能缺少就是数据集

**Reply by leo852** [reply]
什麼特別的要求倒是沒有，就是看看會不會支持廣東話，因為廣東話和普通話的用字有很多不同，發音也不是像普通話那樣只有 4聲，廣東話有 9聲 6調，會比較複雜

**Comment by chairmanqin** [comment]
请问是否计划支持AMD显卡训练？

**Reply by kslz** [reply]
A卡应该装linux再装ROCm就能直接跑

**Comment by 01miaom** [comment]
请问会增加对wenetspeech数据集的支持吗？

wenetspeech包含了大约10000小时的中文语音，数据集链接：

`https://github.com/wenet-e2e/WenetSpeech`

**Reply by babysor** [reply]
该数据集主要是asr用的，放在TTS里面需要做大量的清理工作，而且很不适用于上一代的TTS模型，后面可能会先进行模型升级，才能支持wenetspeech这种数据集

**Reply by 01miaom** [reply]
> 该数据集主要是asr用的，放在TTS里面需要做大量的清理工作，而且很不适用于上一代的TTS模型，后面可能会先进行模型升级，才能支持wenetspeech这种数据集

感谢回复，数据集里的错误确实挺多的，但是数据集的多样性也更好一点，我看到有人在valle上用这个数据跑了非常惊艳的结果。
如果我只用数据集里带高质量标注的数据进行训练也需要更改模型吗，我大致应该对TTS做什么样的修改？

**Reply by babysor** [reply]
是的，vall-e对数据集质量要求会低不少。数据集里面的高质量标注不代表高质量发音，所以会有比较困难的数据清理工作。

**Comment by YIZXIY** [comment]
有计划支持扩散模型吗，看隔壁sovits用的不错，眼馋
